client/my-sites/post-relative-time-status/style.scss

```diff
@@ -1,18 +1,18 @@
 .post-relative-time-status {
-	.time,
-	.status {
+	.post-relative-time-status__time,
+	.post-relative-time-status__status {
 		display: inline-block;
 		margin-right: (11 / 12) * 1em;
```

Comment:
Nevermind, on a closer look, appears the styles were to apply some spacing between the time and trash status shown here:

![image](https://cloud.githubusercontent.com/assets/1779930/16897874/3023cb6a-4b75-11e6-9615-40d446e596aa.png)